docs/book/src/plugins/grafana-v1-alpha.md

```diff
@@ -0,0 +1,96 @@
+# Grafana Plugin
+
+The Grafana plugin provides the Grafana manifest that you can use to visualize your operator status easily.
```

Comment:
wdyt about we make a little bit more clear
```suggestion
The Grafana plugin is an Optional plugin that can be used to scaffold Grafana Dashboards to allow you to check out the default metrics which are exported by projects using [controller-runtime][controller-runtime]
```
we need to add the alias at the bottom.